Engines/Wine/Verbs/mfWorkaround/script.js

```diff
@@ -0,0 +1,113 @@
+const Resource = include("utils.functions.net.resource");
+const { Extractor } = include("utils.functions.filesystem.extract");
+const { ls, cp } = include("utils.functions.filesystem.files");
+//const { getGithubReleases } = include("utils.functions.net.githubreleases"); commented out because it's useless until z0z0z will use releases
+
+const Optional = Java.type("java.util.Optional");
+const Regedit = include("engines.wine.plugins.regedit");
+const Regsvr32 = include("engines.wine.plugins.regsvr32");
+const OverrideDLL = include("engines.wine.plugins.override_dll");
+
+/**
+ * Verb to install Media Foundation workaround
+ * see: https://github.com/z0z0z/mf-install
+ */
+class mfWorkaround {
+    constructor(wine) {
+        this.wine = wine;
+    }
+
+    /**
+     * Sets the used mfWorkaround version
+     *
+     * @param {string} mfWorkaroundVersion The version of mfWorkaround to downlaod
+     * @returns {mfWorkaround} The mfWorkaround object
+     */
+    withVersion(mfWorkaroundVersion) {
+        this.mfWorkaroundVersion = mfWorkaroundVersion;
+
+        return this;
+    }
+
+    go() {
+        const wizard = this.wine.wizard();
+        const prefixDirectory = this.wine.prefixDirectory();
+        const system32directory = this.wine.system32directory();
+        const system64directory = this.wine.system64directory();
+
+        //commented out because it's useless until z0z0z will use releases
+        /*if (typeof this.mfWorkaroundVersion !== "string") {
+            const versions = getGithubReleases("Kron4ek", "FAudio-Builds", wizard);
+            this.mfWorkaroundVersion = versions[0];
+        }*/
+
+        const setupFile = new Resource()
+            .wizard(wizard)
+            .url(
+                `https://codeload.github.com/z0z0z/mf-install/zip/master`
+            )
+            .name(`mf-install-master.zip`)
+            .get();
+
+        new Extractor()
+            .wizard(wizard)
+            .archive(setupFile)
+            .to(`${prefixDirectory}/mfWorkaround/`)
+            .extract();
+
+        const mfWorkaroundDir = `${prefixDirectory}/mfWorkaround/mf-install-master`;
+
+        ls(`${mfWorkaroundDir}/syswow64`).forEach(file => {
+            if (file.endsWith(".dll")) {
+                cp(`${mfWorkaroundDir}/syswow64/${file}`, system32directory);
+
+                new OverrideDLL(this.wine).withMode("native", [file]).go();
+            }
+        });
+
+        if (this.wine.architecture() == "amd64") {
+            ls(`${mfWorkaroundDir}/system32`).forEach(file => {
+                if (file.endsWith(".dll")) {
+                    cp(`${mfWorkaroundDir}/system32/${file}`, system64directory);
+
+                    new OverrideDLL(this.wine).withMode("native", [file]).go();
+                }
+            });
+        }
+
+        const regeditFile = `${prefixDirectory}/mfWorkaround/mf-install-master/mf.reg`;
+        new Regedit(this.wine).patch(regeditFile);
```

Comment:
Try `new Regedit(this.wine).open(regeditFile);`

Reply:
Tried that. It did not fix the issue.

Reply:
`patch` is definitely wrong. See my argument [above](https://github.com/PhoenicisOrg/scripts/pull/1149#issuecomment-569492163).

What "error" do you get when you use `open`?

Reply:
Thats the thing. No error in terminal window in either `patch` or `open`.

Reply:
One parameter should work (the type can be `undefined`). I think it should be
```Javascript
new Regedit(this.wine).patch(cat(regeditFile));
```
`Regedit#patch` writes the `patchContent` to a file. So in your case, it tries to patch the registry with the string "${prefixDirectory}/mfWorkaround/mf-install-master/mf.reg" instead of the content of that file.

Reply:
Yes this is basically the goal. For now I think it is more important to get it building with the least amount of effort. For now I think we should simply [ignore](http://junit.sourceforge.net/javadoc/org/junit/Ignore.html) the test or remove it completely. Most likely I will not be able to do this today.

Reply:
I don't see the difference to using `open`? The point is that the initial start consists of a regedit file which we want to `open` with regedit. `patch` takes a regedit content string, writes it to a temporary file and opens it at the end. This is basically `open` with an additional previous I/O call

Reply:
@plata this is weird. The part of .reg file that was meant to be in `HKEY_LOCAL_MACHINE` is nowhere in registry editor but everything after that is there.

Reply:
@madoar you are right, this should be the same as `open`.
@Zemogiter do you get the same result if you run regedit manually (on the command line)?

Reply:
@madoar yes

Reply:
In this case it cannot be a Phoenicis issue. Maybe something is wrong in the file?

Reply:
@plata just tried it on plain Wine and the whole .reg file is in the registry editor.

Reply:
Have you tried other scripts that make use of `Regedit`? Do they work for you?

Reply:
@madoar yes they do

Reply:
Strange. If Phoenicis use the same command line as plain Wine, how can it be different...